doc/lsst.ctrl.bps/quickstart.rst

```diff
@@ -455,6 +455,36 @@ Supported settings
     * TRANSFORM = Output QuantumGraph files after creating GenericWorkflow.
     * PREPARE = QuantumGraph files are output after creating WMS submission.
 
+**saveClusteredQgraph**
+    A boolean flag.  If set, BPS will save serialized clustered quantum
+    graph to a file called ``bps_clustered_qgraph.pickle`` using
+    Python's `pickle`_ module.  The file will be located  the submit
+    directory.  By default, it is set to false.
+
+    Setting it to true will significantly increase memory requirements
+    when submitting large workflows as `pickle`_ constructs a complete
+    copy of the object in memory before it writes it to disk. [`ref`_]
+
+**saveGenericWorkflow**
+    A boolean flag.  If set BPS will save serialized generic workflow
```

Comment:
The other "If set" have comma afterwards.  (as mentioned above I would say "If set to true,")